core/trino-web-ui/src/main/resources/webapp-preview/src/components/dashboard.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Empty } from "@douyinfe/semi-ui";
```

Comment:
why not material design? it's more established than semi-ui

Reply:
cc @mosabua

Reply:
semi design only because its been used in trino-gateway. But I’m happy to use something else, including material-ui as well as I’m not the greatest fan of semi ui:) I leave it you to decide

Reply:
Me neither. I think we should stick to the material-ui

Reply:
ok I will rework this PR and let you know when it’s ready to review again

Reply:
Let's wait for the @mosabua and maybe @electrum to chime in. Personally I have a gut feeling that using an industry standard library is more appealing for other contributors as the entry bar is much lower for the things that you already known. Learning a new UI library shouldn't be a goal :)

Reply:
I am also inclined to go with material but I am not a webdev any more... @colebow has more experience.

The fact that semi is used in the Trino Gateway UI is something we should keep in mind but it should not force us. Ideally we want to be able to reuse components across at some stage .. but we are not there yet. 

Also .. if material proofs to be a good choice here.. we can also refactor the Trino Gateway UI if we want.

Reply:
@wendigo @mosabua it seems Material UI is the clear winner. I've updated the PR to use MUI and refreshed the screenshots above. Additionally, I made a minimalistic theme adjustment to align the basic styles with the Trino website and documentation.